delivery_driver/views/delivery_carrier.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Moduon Team S.L.
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0) -->
+<data>
```

Comment:
Please add <odoo> as root

Reply:
I don't understand you. What do you mean?

Reply:
Github obfuscated my comment 😅

Reply:
Ok, no problem. It's changed

Reply:
@rousseldenis can you review again, please?
Thanks 😊